lang/en/docs/properties/data/list.md

```diff
@@ -416,28 +428,16 @@ The P norm measures how homogeneous a material is in terms of its chemical compo
 {!esse/example/properties_directory/structural/p-norm.json!}
 ```
 
-### [Point Group Symmetry](../../properties-directory/structural/point-group.md)
+### [Symmetry](../../properties-directory/structural/symmetry.md)
 
-The Point Group of the structure, indicating the symmetry elements that it contains, is also available. 
+The symmetry of the structure, indicating the point group and space group.
 
 ```json tab="Schema"
-{!esse/schema/properties_directory/structural/symmetry_symbol_point_group.json!}
```

Comment:
It's OK that we did this change for now, but we do have `symmetry_symbol_point_group` and `symmetry_symbol_space_group` as the working prototype in ESSE: https://github.com/Exabyte-io/esse/pull/198. Storing simple strings makes things easier and we should probably continue in that direction.